<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm</class>
 <widget class="QWidget" name="LoginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Giriş Yap</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <spacer name="topSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="loginFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="styleSheet">
      <string>background-color: #F5F7FA; border-radius: 24px; border: 1px solid #B2BABB; padding: 32px;</string>
     </property>
     <layout class="QVBoxLayout" name="frameLayout">
      <item>
       <widget class="QLabel" name="loginTitleLabel">
        <property name="text">
         <string>Giriş Yap</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
        </property>
        <property name="styleSheet">
         <string>font-size: 24px; font-weight: bold; color: #2E4053; margin-bottom: 24px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="usernameLineEdit">
        <property name="placeholderText">
         <string>Kullanıcı Adı</string>
        </property>
        <property name="styleSheet">
         <string>padding-left: 32px; height: 32px; border-radius: 12px; border: 1px solid #B2BABB; font-size: 16px; margin-bottom: 16px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwordLineEdit">
        <property name="placeholderText">
         <string>Şifre</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="styleSheet">
         <string>padding-left: 32px; height: 32px; border-radius: 12px; border: 1px solid #B2BABB; font-size: 16px; margin-bottom: 16px;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="errorLabel">
        <property name="text">
         <string></string>
        </property>
        <property name="styleSheet">
         <string>color: #E74C3C; font-size: 14px; margin-bottom: 8px;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loginButton">
        <property name="text">
         <string>Giriş</string>
        </property>
        <property name="styleSheet">
         <string>QPushButton { background-color: #2980B9; color: white; border-radius: 16px; padding: 10px 0; font-size: 16px; font-weight: bold; } QPushButton:hover { background-color: #1ABC9C; }</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="registerLinkButton">
        <property name="text">
         <string>Hesabın Yok mu? Kayıt Ol</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string>QPushButton { color: #2980B9; background: transparent; font-size: 14px; text-decoration: underline; } QPushButton:hover { color: #1ABC9C; }</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="backButton">
        <property name="text">
         <string>Geri Dön</string>
        </property>
        <property name="styleSheet">
         <string>QPushButton { background-color: #B2BABB; color: #2E4053; border-radius: 12px; padding: 8px 0; font-size: 14px; margin-top: 12px; } QPushButton:hover { background-color: #85929E; }</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="bottomSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 